src/main.js

```diff
@@ -53,7 +53,8 @@ const config = {
   oauthTokenURL: expectEnvVariable('OAUTH_TOKEN_URL'),
   oauthClientID: expectEnvVariable('OAUTH_CLIENT_ID'),
   oauthClientSecret: expectEnvVariable('OAUTH_CLIENT_SECRET'),
-  serverRootURL: process.env.SERVER_ROOT_URL || 'http://localhost:' + (process.env.PORT || '3000')
+  serverRootURL: process.env.SERVER_ROOT_URL || 'http://localhost:' + (process.env.PORT || '3000'),
+  cloudFoundryAPI: expectEnvVariable('API_URL')
```

Comment:
Before reading this diff I followed the README and got this error:

```
[2018-03-06T10:55:20.412Z] ERROR (57651 on gds3824.local): Expected environment variable "API_URL" to be set.
```

Could you add it to the README please?

Reply:
Good shout, will do!